config/ecs/config.go

```diff
@@ -6,10 +6,8 @@ package ecs
 
 import (
 	"context"
-	"path/filepath"
-	"strings"
-
 	"github.com/pkg/errors"
+	"path/filepath"
```

Comment:
I think it's the ordering of the imports that the linter is complaining about. If you view the output of the lint job (or run `make lint` locally) it should give you a command to run to make it do the right thing. I bet there's a makefile command for it too, but I haven't been able to get that working locally yet.